getsolus/packages
# dnscrypt-proxy-gui: Update to 1.24.20

**Summary**

- Changelog can be found [here](https://raw.githubusercontent.com/F1ash/dnscrypt-proxy-gui/1.24.20/Changelog)
- Add `homepage` key to `package.yml` (Part of getsolus/packages#411)
- Add appstream metainfo (Part of getsolus/packages#1389)
- Add monitoring.yml

**Test Plan**

- Launch the app (Tested on Plasma VM)
- Verified the added homepage loads information about the package
- Verify metainfo with `appstream-builder --packages-dir=. --include-failed -v`

**Checklist**
